scripts/generate_client_publish_job.py

```diff
@@ -0,0 +1,94 @@
+import argparse
+import os, subprocess
+import yaml
+
+WORKSPACE = os.environ.get("WORKSPACE")
+if WORKSPACE is None:
+    raise RuntimeError("provide the WORKSPACE variable!")
```

Comment:
(pre-existing to your changes)

We don't need `WORKSPACE`, as there is anything from there to be used except for `integration` directory. We could just define `INTEGRATION_DIR` and look for the release tool there.

(pre-existing to your changes too)

Or... even better... can't we just `git clone` integration repo from here into a tmp dir? As it is right now is kind of an undocumented requirement.

Reply:
👌 - also makes the generating jobs immediately 2 lines shorter 🎉